drivers/hmis/app/models/hmis/form/instance_enrollment_match.rb

```diff
@@ -29,7 +29,7 @@ def initialize(instance:, enrollment:)
     when ALL_MATCH
       true
     when HOH_AND_ADULTS_MATCH
-      enrollment.head_of_household? && enrollment.adult?
+      enrollment.head_of_household? || enrollment.adult?
```

Comment:
suggest adding clarify comment:
```suggestion
      #HoH and Adults means that the form should be collected for the HoH AND and adults in the household
      enrollment.head_of_household? || enrollment.adult?
```